_posts/2022-04-04-grant-announcements.md

```diff
@@ -0,0 +1,182 @@
+---
+title: "Announcing Brink's Latest Round of Developer Grants"
+permalink: /blog/2022/04/04/grantees/
+layout: post
+author: brink
+name: "brink"
+alt: Brink announces latest round of developer grants
+category: "News"
+description: Brink announces the latest round of developer grants.
+---
+
+Brink is proud to announce the recipients of its latest round of open source developer grants.
+After an open grant application process, the [Brink board of directors][brink
+board] has selected four developers who have demonstrated continued commitment to
+open source Bitcoin development. The grant recipients are:
+
+## "0xB10C"
+
+<img src="/assets/images/0xb10c.png" alt="Picture of 0xb10c" style="float:left;border:1px solid
+darkgray;margin-right:25px;width:150px;" />
+
+0xB10C writes of his Bitcoin experience and goals:
+
+I am a Bitcoin developer curious about what is happening on the Bitcoin
+network. Over the past years, I've built tools to visualize and observe, for
+example, the Bitcoin mempool, mining pool transaction selection, and other
+protocol level statistics.
+
+After completing the Chaincode Labs Summer Residency seminars in 2019, I worked
+as a freelance developer for ShiftCrypto and CoinMetrics. At CoinMetrics, I
+helped bootstrap [Farum][farum], a real-time network and attack monitoring suite for
+Bitcoin. My work includes patching Bitcoin Core to expose a real-time interface
+for changes to the node's mempool and chain view, along with software to monitor
+the mining jobs of hundreds of Stratum servers to detect, for example, network
+splits early.
+
+I decided to focus entirely on open source projects in early 2021 and accepted
+a one-year grant offer from Coinbase. During this year, I built
+[miningpool-observer][miningpool-observer], mentored two [Summer of Bitcoin][sob] mentees, introduced
+tracepoints for [Userspace, Statically Defined Tracing to Bitcoin Core][usdt], started
+working on making regular reorgs on the Signet test network a reality, and
+published an open-content blog for Bitcoin developers called
+[bitcoin-dev.blog][bitcoin dev blog].
+
+With support from Brink in 2022, I plan to continue work on tracepoints in
+Bitcoin Core, including interface tests, review, and the addition of further
+tracepoints. Another goal is to launch a Signet with regular reorgs and a tool
+to explore reorgs visually. After flooding attacks on the Bitcoin P2P network in
```

Comment:
```suggestion
to explore reorgs visually. After observing flooding attacks on the Bitcoin P2P network in
```

I found this sentence ambiguous---it sounded like 0xB10C *caused* the flooding attacks.  Hopefully this change clarifies that he merely observed them.